languages/csharp/Bitwarden.Sdk/ProjectsClient.cs

```diff
@@ -0,0 +1,74 @@
+﻿namespace Bitwarden.Sdk;
+
+public class ProjectsClient
+{
+    private readonly CommandRunner _commandRunner;
+
+    internal ProjectsClient(CommandRunner commandRunner)
+    {
+        _commandRunner = commandRunner;
+    }
+
+    public ProjectResponse Get(Guid id)
+    {
+        var command = new Command() { Projects = new ProjectsCommand { Get = new ProjectGetRequest { Id = id } } };
+        var result = _commandRunner.RunCommand<ResponseForProjectResponse>(command);
+
+        if (result is { Success: true }) return result.Data;
```

Comment:
Please ensure control blocks always use brackets.
```suggestion
        if (result is { Success: true }) { return result.Data; }
```

Or 

```suggestion
        if (result is { Success: true }) {
          return result.Data;
        }
```

Reply:
Used `if (result is { Success: true }) { return result.Data; }` everywhere